apps/frontend/src/shared/components/ui/ImageSlider.tsx

```
@@ -0,0 +1,170 @@
+    <div className={cn('relative overflow-hidden select-none', className)} ref={containerRef}>
+      {/* Image Container */}
+      <div
+        className="relative w-full h-full cursor-grab active:cursor-grabbing"
+        onMouseDown={handleMouseDown}
+        onMouseMove={handleMouseMove}
+        onMouseUp={handleMouseUp}
+        onMouseLeave={handleMouseLeave}
+        onTouchStart={handleTouchStart}
+        onTouchMove={handleTouchMove}
+        onTouchEnd={handleTouchEnd}
+      >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)